homework.md

```diff
@@ -86,19 +88,19 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+A collaborative place for us to store and share our code.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Save your changes in Github
 
```

Comment:
Github is the place where the remote repositories are stored. Does `commit` save changes to remote repositories?